src/osip-core/src/main/java/edu/kit/pse/osip/core/utils/language/Translator.java

```diff
@@ -46,8 +51,12 @@ public String getString(String key) {
     public void setLocale(Locale locale) {
         if (locale == null) {
             throw new NullPointerException();
+        } 
+        try {
+            bundle = ResourceBundle.getBundle("Bundle", locale);
+        } catch (MissingResourceException e) {
+            e.printStackTrace();
```

Comment:
Das Grundproblem bleibt: Wenn es das Bundle nicht gibt, wird `bundle` auf `null` gesetzt, ohne dass der Aufrufer von `setLocale()` etwas davon mitbekommt. Wenn er dann `getString()` aufruft, dann bekommt er sofort eine `NullPointerException,` da `bundle == null`. Darum schlage ich folgendes vor:

- In `setLocale`: Wenn es das Bundle nicht gibt, dann benutzte System.err.println() um eine Fehlermeldung auszugeben, anstatt den ganzen Stacktrace auszugeben (sieht einfach schöner aus für den Benutzer)

- In `getString`: Wenn `bundle == null`, dann gebe einfach key zurück